src/game_api/search.cpp

```diff
@@ -2034,6 +2034,50 @@ std::unordered_map<std::string_view, AddressRule> g_address_rules{
             .offset(0x21)
             .at_exe(),
     },
+    {
+        "unload_layer"sv,
+        // bp on destroy entity, leave level, it's third in stack or something
+        PatternCommandBuffer{}
+            .find_inst("49 89 cc 8b 41 18 85 c0 74 0b 49 8b 74 24 08"_gh)
+            .at_exe()
+            .function_start(),
+    },
+    {
+        "init_layer"sv,
+        // TODO
+        PatternCommandBuffer{}
+            .from_exe_base(0x228b58f0),
+    },
+    {
+        "dead_players"sv,
+        // TODO
+        PatternCommandBuffer{}
+            .from_exe_base(0x22c061d0),
```

Comment:
And this is screen level `v1` virtual

Reply:
Also death screen is `v4`
i should probably look more into those

Reply:
btw. screen in state, not in gamemaganer, we should really make some clear distinctions hmm

Reply:
I was just going to make patterns, but how is using hardcoded VTABLE_OFFSETs any better than using hardcoded RVAs though?

Reply:
vtable offsets i can update with one click in the plugin

Reply:
So it's still worse than pattern, assuming the pattern didn't break on update, and better than pattern if the pattern did break on update...

Reply:
and both are better than harcoded offsets :)

i don't mind patterns, thou using a vtable offset makes it also clear where to look for it if it breaks

Reply:
Also the vtable offsets don't break every update